src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -351,6 +351,8 @@ public async Task<Manifests> UpdateManifestsAutonomously(Manifests manifests)
                 Logger.ErrorLocalized(nameof(Resources.NewInstallerUrlMustMatchExisting_Message));
                 installerMatchException.MultipleMatchedInstallers.ForEach(i => Logger.ErrorLocalized(nameof(Resources.UnmatchedInstaller_Error), i.Architecture, i.InstallerType, i.InstallerUrl));
                 installerMatchException.UnmatchedInstallers.ForEach(i => Logger.ErrorLocalized(nameof(Resources.MultipleMatchedInstaller_Error), i.Architecture, i.InstallerType, i.InstallerUrl));
+                Console.WriteLine();
+                Logger.ErrorLocalized(nameof(Resources.UseOverrides_ErrorMessage));
 
                 if (installerMatchException.IsArchitectureOverride)

```

Comment:
The placement here will suggest to the user that they should use an override if the installers fail to match. The if statement below that will also show a warning to the user if the architecture override does not work which is redundant. 

I would change this to `Logger.WarnLocalized` and put it in an else clause so that it only warns the user if they do not provide an architecture override.